src/radio/styled-components.js

```diff
@@ -205,12 +216,31 @@ export const Input = styled('input', {
 });
 
 export const Description = styled<StylePropsT>('div', props => {
+  const {$theme, $align} = props;
+  const isHorizontal = $align === 'horizontal';
+  let marginBefore = 'Left';
+  if ($theme.direction === 'rtl') {
+    if (marginBefore === 'Left') {
+      marginBefore = 'Right';
+    } else if (marginBefore === 'Right') {
+      marginBefore = 'Left';
+    }
+  }
+  let marginAfter = 'Right';
+  if ($theme.direction === 'rtl') {
+    if (marginAfter === 'Left') {
```

Comment:
This could be done in a utility function or the variable could be assigned in a slicker way but I like the clarity and simplicity of rtl swap. Let me know if we want this to be more terse though and I can rewrite it.

Reply:
actually now that I see it written out I'm going to go with the shorter version haha